akka-cluster/src/main/scala/akka/cluster/Gossip.scala

```diff
@@ -22,6 +22,8 @@ private[cluster] object Gossip {
   def apply(members: immutable.SortedSet[Member]) =
     if (members.isEmpty) empty else empty.copy(members = members)
 
+  def vclockName(node: UniqueAddress): String = s"${node.address}-${node.longUid}"
+
```

Comment:
Moved it here because it didn't make sense to have a dependency back to `ClusterDaemon` from `Gossip`